### ISSUE TITLE
Update dependencies and prune lock files

### ISSUE THREAD

**Issue by novellizator** [salsita/foosball-rating#120]
https://github.com/salsita/foosball-rating/network/alerts points to the problems and they are two fold.

- [x] deps need to be updated

- [x]  we seem to be using both package-lock and yarn-lock. Let's delete one of them (and put it in gitignore) and be consistent

**Comment by littlewhywhat** [comment]
so at the current moment there are alerts regarding minimist 0.0.8.
For root package.json:
```
├─┬ eslint@6.8.0
│ └─┬ mkdirp@0.5.1
│   └── minimist@0.0.8 
```
comes from eslint 6.8.0 and it's the latest release of eslint - it actually uses this old version of mkdirp https://github.com/eslint/eslint/blob/v6.8.0/package.json#L75

should be resolved with this one https://github.com/eslint/eslint/issues/13050 - then we can update eslint.

**Comment by littlewhywhat** [comment]
for backend/package.json:
```
└─┬ node-pg-migrate@4.2.2
  └─┬ mkdirp@0.5.1
    └── minimist@0.0.8 
```
Used by node-pg-migrate as you see but actually this is strange for me because the package uses ~1.0.0 of mkdirp - https://github.com/salsita/node-pg-migrate/blob/master/package.json#L48 that doesn't use minimist at all https://github.com/isaacs/node-mkdirp/blob/v1.0.0/package.json... I tried to reinstall but still I get old mkdirp in package-lock.json.

**Comment by littlewhywhat** [comment]
for node-pg-migrate - the problem is that in tgz package we still use mkdirp 0.5.0 therefore npm keeps installing it

**Comment by littlewhywhat** [comment]
still need to run npm audit fix in frontend and root folders

**Comment by littlewhywhat** [comment]
fixed by #134 and #123
